### PR TITLE
Increment master

### PR DESCRIPTION
**Pull request by inlined**
Add back the increment operator.

### CODE REVIEW

File: packages/database/src/api/Database.ts [hunk 1]
```diff
@@ -1,6 +1,6 @@
 /**
  * @license
- * Copyright 2017 Google Inc.
+ * Copyright 2017 Google LLC
```
Comment (schmidt-sebastian) [inline packages/database/src/api/Database.ts]: Per go/copyright, we are not supposed to update copyright notices in old files.

File: packages/database/test/servervalues.test.ts [hunk 2]
```diff
@@ -0,0 +1,151 @@
+/**
+ * @license
+ * Copyright 2019 Google LLC
```
Comment (schmidt-sebastian) [inline packages/database/test/servervalues.test.ts]: 2020?

File: packages/database/src/api/Database.ts (hunk 1 above)
Comment (hsubox76) [reply packages/database/src/api/Database.ts]: This is the PR that causes this update: https://github.com/firebase/firebase-js-sdk/pull/2665

From the description it sounds like it is intentional that it should update changed files and I think that's what go/copyright says (retain oldest year, change Inc. to LLC)

**Comment by inlined** [comment]
@schmidt-sebastian Looks like this is ready to merge?

**Comment by inlined** [comment]
@Feiyang1 can you approve the version bump of the emulator? It's necessary to get the server-side changes for increment operations.

**Pull request merged**